### DISCUSSION TITLE
Change glow around notes

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#13104] [Ideas]
**Describe the new feature:** Now, a friend of mine thought of this and fiddled with Stable a bit to demonstrate it and I thought it would fit really well with Laser's aesthetics.
The idea is to change the glow around notes to make it more gentle and spread out.

**Proposal designs of the feature:** 
![image](https://user-images.githubusercontent.com/15932372/72210293-5c476b80-34b9-11ea-9b00-3b46b39835af.png)

**Comment by airone01** [comment]
That's actually super cool !

I noticed that the join between the two colors is not exactly between the two notes (or I'm just colorblind idk).
I think this needs some experimenting in-game, to find out exactly at which distance from the notes the glowing needs to change. It may defer depending on track speed too.
